s4e-backend/src/main/java/pl/cyfronet/s4e/geoserver/sync/GeoServerSynchronizer.java

```diff
@@ -18,14 +19,25 @@
 @Slf4j
 public class GeoServerSynchronizer {
     private final SldStyleRepository sldStyleRepository;
+    private final ProductRepository productRepository;
     private final PRGOverlayRepository prgOverlayRepository;
 
     private final GeoServerService geoServerService;
 
+    @Transactional
+    public void synchronizeStoreAndMosaics() {
+        log.info("Creating stores and mosaics");
+        for (val product : productRepository.findAll()) {
+            if (!geoServerService.layerExists(product.getName().toLowerCase())) {
+                geoServerService.addStoreAndLayer(product);
+            }
+        }
+    }
+
     @Transactional
     public void synchronizeOverlays() {
         log.info("Creating styles");
-        for (val sldStyle: sldStyleRepository.findAll()) {
+        for (val sldStyle : sldStyleRepository.findAll()) {
```

Comment:
Why break formatting here?

Reply:
for this type its default formatting, there are multiple examples in our code, commited also by you :)
for me this way is cleaner to read. also it was me doing ctrl+alt+l, not explicity changing that line.